summary_workflows/identification/specification/I_benchmark_specification.md

```diff
@@ -130,6 +147,32 @@ Ranking: For each tool, the area under the curve is calculated. The best perform
 
 Note: 2D line plot is not supported in OpenEBench yet. If it's not implemented, the data should be visualised outside of OpenEBench.
 
+2. Grouped **bar plot** visualizing **proportions of PAS assigned to different genomic features** in one plot.
+
+**metric** - proportions of PAS assigned to different genomic features. Multiple bars corresponding to multiple featues are plotted using different colors.
```

Comment:
This is more or less the plot format I had in mind:

![example_plot](https://user-images.githubusercontent.com/30384499/167836894-4939cf03-4d7b-4923-a6d1-9ea2499d2738.png

In this case the feature assignments don't have to add up to 100%, so assignment to overlapping features is less of a problem. I'd maybe add a note saying that this is the case. 
But I agree the features might need changing.
Maybe something like this:
- 3'-terminal exon
- exon (excluding 3'-terminal exon?)
- intron
- everything else

Based on your comments on 3'UTR metric, I'd also add ground truth to the plots along the benchmarked tools.

Reply:
Makes sense with the example plot thank you! I like the categories you suggest, would definitely have the 'exon' category exclude the 3'-terminal exon. I think it would be nice to have an additional category that is 'downstream of 3'-terminal exon' within an arbitrary distance (e.g. 1kb, 5kb etc.). I don't think that is essential though, 'intergenic' for the rest would cover just fine for now.

Reply:
Do we still need to do something here?

Reply:
I've changed the features to:
- 3'-terminal exons
- exon (excluding 3'-terminal exon?)
- introns
- intergenic region within 1kb downstream of 3'-terminal exon
- intergenic region more than 1kb downstream of 3'-terminal exon

This is an optional plot, so if we think something else might be more useful later on, we can still change it.

Reply:
This is ok on my end now!